### PR TITLE
fix(tests): Fixing Onboarding Acceptance Snapshot

### PR DESCRIPTION
**Pull request by ehfeng**
Hard setting public and private keys so Percy doesn't surface them as diffs every time.

### CODE REVIEW

**Review by dcramer — APPROVED** [review]
lgtm but you need to change private_key -> secret_key

aside i would format like:

```
...objects.filter(
    foo=bar,
).update(
    biz=baz,
)
```

**Comment by ehfeng** [comment]
Huh, I just kind of assume yapf would auto-format it for me.

Fixed.

File: tests/acceptance/test_onboarding.py [hunk 1]
```diff
@@ -21,10 +24,10 @@ def setUp(self):
             teams=[self.team],
         )
         self.login_as(self.user)
-        self.path = '/onboarding/%s/' % self.org.slug
 
-    def test_onboarding(self):
-        self.browser.get(self.path)
+    @mock.patch('uuid.uuid4', return_value=UUID('031667ea1758441f92c7995a428d2d14'))
```
Comment (ehfeng) [inline tests/acceptance/test_onboarding.py]: @mattrobenolt I tried following the mocking of `uuid4`, but it's not '031667ea1758441f92c7995a428d2d14' as I would expect.

image

I tried running...
```python
@mock.patch('uuid.uuid4', return_value=UUID('031667ea1758441f92c7995a428d2d14'))
def test(uuid4):
  print(uuid4().hex)

test()
```
...and that returns the expect value. Any ideas?

Comment (mattrobenolt) [reply tests/acceptance/test_onboarding.py]: Yeah, you need to patch the uuid that's imported into the package.

So your mock path needs to be something like `sentry.models.projectkey.uuid4`

**Pull request merged**